src/main/resources/assets/ars_nouveau/lang/zh_cn.json

```
@@ -561,11 +562,25 @@
 	"ars_nouveau.wixie.crafting": "合成：",
 	"ars_nouveau.wixie.need_mana": "需要魔力。",
 	"ars_nouveau.wixie.needs": "需要原料：",
-	
+	"ars_nouveau.wixie.needs_storage": "需要：更多药水存储空间。",
 	"ars_nouveau.true": "是",
 	"ars_nouveau.false": "否",
   
 	"ars_nouveau.spell_hud.crafting_mode": "法术制作模式",
 	
-	"ars_nouveau.spell.no_mana": "魔力不足。"
+	"ars_nouveau.spell.no_mana": "魔力不足。",
+	"block.ars_nouveau.potion_jar": "药水罐",
+	"block.ars_nouveau.potion_melder": "药水混合器",
+	"item.ars_nouveau.potion_flask_extend_time": "增强药水烧瓶：延长时效",
+	"item.ars_nouveau.potion_flask_amplify": "增强药水烧瓶：强化",
+	"tooltip.potion_flask_extend_time": "将药水的持续时间延长 50%。",
+	"tooltip.potion_flask_amplify": "提升药水等级，但会减少持续时间。",
+	"tooltip.potion_flask": "最多能容纳 8 份药水。",
+	"ars_nouveau.locked": "已锁定",
+	"ars_nouveau.unlocked": "已解锁",
+	"ars_nouveau.page4.wixie_charm": "薇克精会使用附近的药水罐和物品自动合成药水。合成中需要的水会由薇克精自己提供。如果合成时需要另一种药水作为基底，薇克精则会从附近药水罐抽取基底药水。合成完成时，薇克精会向附近药水罐输出 3 剂药水。要开始尝试合成，首先放下一个空药水罐，使用粗制的药水右击薇克精坩埚，然后在附近的箱子提供下界疣。",
+	"ars_nouveau.page5.wixie_charm": "同时使用多个薇克精时，你可能会想将药水罐锁定为存放特定药水。薇克精只会向含有同种药水的药水罐，或者未锁定的空药水罐输出药水。每次合成都会消耗魔力。请参阅药水罐条目以获取更多信息。",
```

|Feedback Priority|
|---|
|🧗‍♀️ Boulder / **Blocking**|

```suggestion
	"ars_nouveau.page5.wixie_charm": "同时使用多个薇克精时，你可能会想将药水罐锁定，使其只存放特定药水。薇克精只会向含有同种药水的药水罐，或者未锁定的空药水罐输出药水。每次合成都会消耗魔力。请参阅药水罐条目以获取更多信息。",
```
疑似语病。